rocketc/rocketc.py

```diff
@@ -71,12 +71,68 @@ class RocketChatXBlock(XBlock, XBlockWithSettingsMixin, StudioEditableXBlockMixi
         scope=Scope.user_state
     )
 
+    emoji = String(
+        display_name="Emoji to grade with",
+        default="",
+        scope=Scope.settings,
+        help="Select the emoji which you want to grade",
+    )
+
+    oldest = DateTime(
+        display_name="Date From",
+        default=None,
+        scope=Scope.settings,
+        help="ISO-8601 formatted string representing the start date of this assignment."
+    )
+
+    latest = DateTime(
+        display_name="Date To",
+        default=None,
+        scope=Scope.settings,
+        help="ISO-8601 formatted string representing the due date of this assignment."
+    )
+
+    target_reaction = Integer(
+        display_name="Target Reaction Count",
+        default=5,
+        scope=Scope.settings,
+        help="Target value in order to achieve a defined grade."
+    )
+
+    graded_activity = Boolean(
+        display_name="Graded Activity",
+        default=False,
+        scope=Scope.settings,
+    )
+
+    weight = Float(
+        display_name="Score",
+        help="Defines the number of points each problem is worth. ",
+        values={"min": 0, "step": .1},
+        default=1,
+        scope=Scope.settings
```

Comment:
what is a problem in this rocket chat context?

Reply:
The problem is get a number of emojis for a specific comment on rocketchat